sandbox-web/src/main/java/com/exadel/sandbox/team5/ReviewRestController.java

```diff
@@ -3,26 +3,40 @@
 import com.exadel.sandbox.team5.entity.Review;
 import com.exadel.sandbox.team5.service.ReviewService;
 import lombok.RequiredArgsConstructor;
-import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
-import org.springframework.web.server.ResponseStatusException;
 
 import java.util.List;
-import java.util.stream.Collectors;
 
 @RestController
-@RequestMapping("/review")
+@RequestMapping("/reviews")
 @RequiredArgsConstructor
 public class ReviewRestController {
     private final ReviewService service;
 
     @GetMapping("/{id}")
-    public List<Review> save(@PathVariable Long id) {
-        if (id != null && service.getById(id) != null) {
-            return service.getAll().stream().filter(entity -> entity.getDiscount().getId().equals(id)).collect(Collectors.toList());
-        } else {
-            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Unknown discount id");
-        }
+    public Review getById(@PathVariable Long id) {
+        return service.getById(id);
+    }
+
+    @GetMapping("/all")
```

Comment:
extra string parameter can removed. GET with review id will try to return exact object without id all objects